src/main/java/com/gojek/beast/sink/bq/handler/impl/OOBErrorHandler.java

```diff
@@ -58,8 +58,8 @@ public Status handleErrorRecords(Map<Record, List<BQInsertionRecordsErrorType>>
         if (shouldBatchFail) {
             //lets not store OOB records as well as the batch contains invalid/schema related errors
             log.info("Batch with records size: {} contains invalid records, marking this batch to fail", records.size());
-            statsClient.gauge("sink.unprocessed.invalid.err.records", invalidRecordsCount);
-            statsClient.gauge("sink.unprocessed.unknown.err.records", unKnownErrorRecordsCount);
+            statsClient.gauge("data.error.records,type=invalid", invalidRecordsCount);
```

Comment:
We should have this block outside of the for loop, lets send 0 if there are no such records.

Reply:
That'd be unwanted metric which we wouldn't need for data validity panel.
Also a `0` count would be emitted most of the time hoarding up influx.